src/routes/FAQ.svelte

```diff
@@ -1,17 +1,18 @@
 <script>
+  import { TranslatedStrings } from "$lib/translations/translations";
   import { Alert, Button } from "flowbite-svelte";
+  import discord from "$assets/images/Discord.png";
```

Comment:
We're using an icon library for this project, for the sake of consistency please use the icon library (fa-icons) instead of a png file. We also want to avoid bloat from unnecessary images.